src/platform/packages/shared/kbn-esql-ast/src/types.ts

```diff
@@ -96,6 +97,11 @@ export interface ESQLAstJoinCommand extends ESQLCommand<'join'> {
   commandType: 'lookup' | 'left' | 'right';
 }
 
+export interface ESQLForkBranch extends ESQLAstBaseItem {
```

Comment:
I'm thinking we may not need this, as we have `ESQLAstQueryExpression`, which represents a query.

If we want a specific type, just for the fork branch it could extend the former:

```ts
interface ESQLForkBranch extends ESQLAstQueryExpression {
  queryType: 'fork_branch';
  commands: Array<ESQLCommand<'where'> | ESQLCommand<'sort'> | ESQLCommand<'limit'>>;
}
```

Reply:
Good point... let me look into this.

Reply:
Done in https://github.com/elastic/kibana/pull/216743/commits/9d774c4035d91f3258be73e80a7a946aa29b1b8e